src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -0,0 +1,79 @@
+package org.opentripplanner.middleware.models;
+
+import org.opentripplanner.middleware.otp.core.api.model.Itinerary;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * A monitored trip represents
+ */
+public class MonitoredTrip extends Model {
+
+    /**
+     * User Id. Id of user monitoring trip.
+     */
+    public String userId;
+
+    /**
+     * Name given to the trip by the user
+     */
+    public String tripName;
+
+    /**
+     * The time at which the trip takes place. This will be in the format HH:mm and is extracted (or provided separately)
+     * to the date and time within the query parameters. The reasoning is so that it doesn't have to be extracted every
+     * time the trip requires checking.
+     */
+    public String tripTime;
+
+    /**
+     * The number of minutes prior to a trip taking place that the status should be checked.
+     */
+    public int leadTimeInMinutes;
+
+    /**
+     * The days on which the trip status should be checked. This will be a text representation of each day of the week.
+     * Enums have not been used because they are not compatible with MongoDB at this point in time. Java's number
+     * representation hasn't been used because it is irrelevant to calling systems.
+     *
+     * Expecting each effected day no more than once with the following representation:
+     *
+     * "monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"
+     */
+    public Set<String> days;
+
+    /**
+     * Specify if the monitored trip should be checked on a federal holiday.
+     */
+    //TODO define federal holiday source!
+    public boolean excludeFederalHolidays;
+
+    /**
+     * Specify if the monitored trip is active. If true, the trip will be checked.
+     */
+    public boolean isActive = true;
+
+    /**
+     * Query params. Query parameters influencing trip.
+     */
+    public String queryParams;
+
+    /**
+     * The trips itinerary
+     */
+    public List<Itinerary> itinerary;
```

Comment:
What was the decision on this @landonreed? Did we decide to focus on one itinerary or multiple?

Reply:
I think it is ok for now because from the UI, we query the itinerary without real-time updates, and `queryParams` contains the index of the active itinerary.

Reply:
This should be a single itinerary that is stored. When we check for differences we want to be checking against a single itinerary.